_security-plugin/index.md

```diff
@@ -8,20 +8,62 @@ redirect_from:
+
+## Features at a glance
+
+The following topics provide a general description of the features that define security in OpenSearch.
+
+### Encryption
+
+Encryption protects both data at rest and in transit. At rest, encryption secures sensitive information that’s kept as stored data. Some examples of stored data include indexes, logs, swap files, automated snapshots, and all data in the application directory. 
+
+Encryption in transit encrypts information moving to, from, and within the cluster. OpenSearch uses the Transport Layer Security (TLS) protocol, which covers both client-to-node encryption (the REST layer) and node-to-node encryption (the transport layer). This combination of in-transit encryption guarantees that both requests to OpenSearch and the movement of information among different nodes is safe from tampering.
+
+You can find out more about configuring TLS in the [Configure TLS certificates](https://opensearch.org/docs/latest/security-plugin/configuration/tls/) section.
+
+### Authentication
+
+Authentication is used to validate the identity of users and works by verifying an end user’s credentials against a backend configuration. These credentials can be a simple name and password, a JSON web token, or a TLS certificate. Once the authentication domain extracts those credentials from a user’s request it can check their validity against the authentication backend.
+
+The backend used for validation can be OpenSearch's built-in internal user database – used for storing user and roles configurations and hashed passwords – or one of a wide range of industry-standard identification protocols such as LDAP, Active Directory, SAML, and OpenID Connect. A common practice is to chain together more than one authentication method to create a more robust defense against unauthorized access. This might involve, for example, HTTP basic authentication followed by a backend configuration that specifies the LDAP protocol. See the [Backend configuration](https://opensearch.org/docs/latest/security-plugin/configuration/configuration/) section to learn more about setting up the backend.
+
+### Access control
+
+Access control (or authorization) generally involves selectively assigning permissions to users that allow them to perform specific tasks, such as clearing the cache for a particular index or taking a snapshot of the cluster. However, rather than assign individual permissions directly to users, OpenSearch manages access control by instead assigning permissions to roles and then mapping these roles to users. For more on setting up these relationships, see [Users and roles](https://opensearch.org/docs/latest/security-plugin/access-control/users-roles/). Roles, therefore, define the actions that users can perform, including the data they can read, the cluster settings they can modify, the indexes to which they can write, and so on. Roles are reusable across multiple users, and users can have multiple roles.
```

Comment:
Link:

```
[Users and roles]({{site.url}}{{site.baseurl}}/security-plugin/access-control/users-roles/)
```

Reply:
I agree, it's too wordy. But I think I can boil this down to ...
"However, rather than assign individual permissions directly to users, OpenSearch assigns these permissions to roles and then maps the roles to users."